js/interface-lists.js

```diff
@@ -510,7 +510,9 @@ function attahObservers() {
   });
 
   Fliplet.Widget.onSaveRequest(function () {
-    $('form').submit();
+    if (dynamicLists.isLoaded) {
+      $('form').submit();
+    }
```

Comment:
@MaksymShokin Looks like you've managed to find the problem and the right point to make changes. However, the method you've used results a **Save & Close** button that doesn't do anything when users click on it. Users would feel that something isn't working.

Instead, I'd recommend running `Fliplet.Widget.complete();` if `!dynamicLists.isLoaded`, which will close the widget interface without saving anything.

Reply:
@tonytlwu Fixed. If everything is fine could you please merge this PR because it holds the PR from this task https://github.com/Fliplet/fliplet-studio/issues/4660.